include/communication/models/communication.h

```diff
@@ -0,0 +1,48 @@
+#ifndef COMMUNICATION_H
+#define COMMUNICATION_H
+
+#include "i_communication.h"
+#include "named.h"
+
+#include <QObject>
+#include <QVariantMap>
+#include <QVector>
+
+#include "i_communication_protocol.h"
+#include "i_link_transceiver.h"
+#include "link_specification.h"
+#include "protocol_specification.h"
+
+namespace md::data_source
+{
+class Communication : public domain::ICommunication
+{
+    Q_OBJECT
+
+public:
+    Communication(const domain::LinkSpecification& linkSpecification,
+                  const domain::ProtocolSpecification& protocolDescription,
+                  ICommunicationProtocol* protocol, const QString& name = QString(),
+                  QObject* parent = nullptr);
+    QString type() const override;
+    bool isConnected() const override;
+    int bytesReceived() const override;
+    int bytesSent() const override;
+
+public slots:
+    void setConnected(bool connected) override;
```

Comment:
What is the difference between setConnected and start/stop?

Reply:
I don't know how exactly I should implement them, but for now start/stop is for creation/destruction and threading; 
connect/disconnect is for opening/closing sockets in the future